tools/update-punycode.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+curl -o "lib/punycode.js" "https://raw.githubusercontent.com/bestiejs/punycode.js/master/punycode.js"
```

Comment:
Better yet, _do_ add `-L` and point the URL to https://github.com/bestiejs/punycode.js/raw/master/punycode.js (which will redirect; githubusercontent.com is the implementation detail at this point).